packages/material-ui/src/Stepper/Stepper.js

```diff
@@ -63,18 +63,20 @@ function Stepper(props) {
       connector: connectorProp,
     };
 
-    const state = {
-      active: false,
-      completed: false,
-      disabled: false,
-    };
+    function getStepState(active, idx) {
+      const stepState = { active: false, completed: false, disabled: false };
```

Comment:
You can write this in a single expression i.e. instead of 
```js
const foo {
  bar: false
}

if (isBar) {
  foo.bar = true
}
```
write 
```
const foo = {
  bar: isBar
}
```